src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -868,6 +868,9 @@ namespace OutputReportTabular {
             // add to the data structure
             curTable = AddMonthlyReport(AlphArray(1), int(NumArray(1)));
             for (jField = 2; jField <= NumAlphas; jField += 2) {
+                if (AlphArray(jField).empty()) {
```

Comment:
Here, however, if the input field is blank, we throw a fatal error.  I confirmed this by editing some test files for the three objects: summary, monthly, and annual.  They all fatal error with this message indicating it was because of a blank report name.  Let me know if that seems reasonable @mjwitte, keeping in mind that we are short on time, and the current behavior is to silently crash with no explanation given.

Reply:
I'd prefer if we could just skip blank names with no error, but if that's too complicated, then this is better than v8.9 behavior.

Reply:
I'm not going to be able to get anything beyond this done for 9.0 myself.  I'll open up a follow-on ticket.